scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -374,14 +371,28 @@ class FossId internal constructor(
             }
         }
 
+    /**
+     * Filter this list of [Scan]s for the repository defined by [url] and an optional [revision] and sorts it by
```

Comment:
This still says "sorts" instead of "sort".

Reply:
Sorry, missed this one. Should be fixed now.